### ISSUE TITLE
Needs formatting

### ISSUE THREAD

**Issue by brs713** [Kritner/KritnerWebsite#55]
https://www.freecodecamp.org/news/docker-compose-nginx-and-letsencrypt-setting-up-website-to-do-all-the-things-for-that-https-7cb0bf774b7e/

Great tutorial, but none of the code is formatted, making it impossible to read at-a-glance.  I took it further than a glance and copied it into a notepad file, then tried to just dump things into online beautifiers, but that only worked for a small percentage of items.

Tested on Windows 10
Chrome - Version 94.0.4606.81 (Official Build) (64-bit)
Firefox - Version 93.0 (64-bit)

**Comment by Kritner** [comment]
Yeah sorry, I haven't had any luck "claiming" my freecodecamp account in order to edit this page - it used to be hosted on medium where the formatting was fine, and was since moved to a separate product.  Here's a link to that same article on my self hosted blog: https://blog.kritner.com/2018/09/19/docker-nginx-and-letsencrypt-website-setup-for-that-sweet-sweet-https/
